_data/sidebars/community_sidebar.yaml

```diff
@@ -11,17 +11,69 @@ entries:
     url: /community.html
     output: web
 
-  - title: Coupled Problems 2025
-    url: /eccomas-coupled-2025.html
+  - title: preCICE workshops
     output: web
+    folderitems:
 
-  - title: preCICE Workshop 2024
-    url: /precice-workshop-2024.html
-    output: web
-  
-  - title: iacm WCCM 2024
-    url: /iacm-wccm-2024.html
+    - title: The preCICE Workshop
```

Comment:
Alternative sidebar names for the page could be "General" or "Overview".
I think the url is good and the title on the page is also independent. But on the sidebar it could look weird to have "The preCICE Workshop" under "preCICE workshops".